src/graphics/graphics.js

```diff
@@ -1071,6 +1071,7 @@ export var UNIFORMTYPE_FLOATARRAY = 17;
 export var UNIFORMTYPE_TEXTURE2D_SHADOW = 18;
 export var UNIFORMTYPE_TEXTURECUBE_SHADOW = 19;
 export var UNIFORMTYPE_TEXTURE3D = 20;
+export var UNIFORMTYPE_VEC4ARRAY = 21;
```

Comment:
If we're exposing VEC4ARRAY, should we do VEC2 + VEC3 too for completeness?

Reply:
good idea, I'll add that.

Reply:
done